index.d.ts

```diff
@@ -0,0 +1,32 @@
+type Mode = "binary" | "metric" | "decimal" | "compatibility";
+
+interface ParseOptions {
```

Comment:
Question: in typescript would this interface be directly used?  (or always inferred) because if you need to use it this should all be documented in the readme above.

Reply:
You would only need to refer to this interface if you wanted to store the options object in a strongly typed variable for processing, rather than inlining the object into the call. Seems innocent but also unlikely. You could use the `Parameters<typeof func>`  type to get the ducktype anyway. I wouldn't export this interface if there's no demand for it.